src/web/theme/ui/templates/Layout/Layout.scss

```diff
@@ -6,7 +6,7 @@ html {
 }
 
 body {
-  color: $fontColor;
+  color: $fontSecondaryColor;
```

Comment:
So the default font color is `$fontSecondaryColor`? It seems weird to me.

Reply:
You don't use primary button every where . Only on important spot where you want the user focus. It's why I decided to place font secondary color on every simple text and keep font primary color to the important text.